test/features/errors/notfound.js

```diff
@@ -66,4 +70,95 @@ describe('404 handling', function() {
             assert.contentType(e, 'application/problem+json');
         });
     });
+    it('should return 404 on deleted revision', function() {
+        return preq.get({ uri: server.config.bucketURL + '/revision/' + revPageDeleted })
+            .then(function() {
+                throw new Error('404 should be returned')
+            })
+            .catch(function(e) {
+                assert.deepEqual(e.status, 404);
+                assert.contentType(e, 'application/problem+json');
+            })
+    });
+    it('should set page_deleted on deleted page', function() {
+        var apiURI = server.config
```

Comment:
Just using the plain URI string seems more sensible here and improves readability